CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- enable smacks in converse conf when websocket is used
```

Comment:
Maybe add a word about  the feature smacks would enable ?
```suggestion
- reconnect converse with bosh when websocket is used through smacks configuration
```

Reply:
that's not `smacks` job to do that. Smacks prevent to lose message when a user is disconnected by hibernating the user session https://modules.prosody.im/mod_smacks

Switching between bosh and websocket is a built-in converse feature that doesn't need extra configuration.

Reply:
Oh, ok sorry.
But I still think we should explain what will change.

Reply:
I should add the link added in my previous comment ?

Reply:
Maybe, yes

Reply:
Done